### DISCUSSION TITLE
Welcome to transcript.fish Discussions!

### DISCUSSION THREAD

**Discussion by noman-land** [noman-land/transcript.fish#31] [Announcements]
## 👋 Welcome!

This is an ongoing, open effort to transcribe every word uttered on the No Such Thing As A Fish podcast. Please feel free to:
  * Ask questions.
  * Share ideas.
  * Make suggestions.
  * Engage with other community members.
  * Welcome others.

The current database of transcriptions is available at: https://media.transcript.fish/db/transcript.db. Feel free to poke around.

**Comment by BoomsplodeKat** [comment]
Hi. I realise this discussion is 3 years old, but the link no longer works. I got Error 404.
<img width="1080" height="2400" alt="Screenshot_20260805_221035_Chrome" src="https://github.com/user-attachments/assets/f0f34298-c3d6-4957-8000-23b5ef172c1a" />

**Comment by noman-land** [comment]
Hi @QuestMaster2-0 thanks for noticing the broken link! I moved the files around since the original message was posted. The new path is https://media.transcript.fish/db/transcript.db.
